mvcc/backend/batch_tx.go

```diff
@@ -166,7 +166,9 @@ func (t *batchTx) commit(stop bool) {
 			return
 		}
 		start := time.Now()
+		// gofail: var beforeCommit struct{}
```

Comment:
It means the failpoint doesn't have any value, as opposed to something like:
```
// gofail: var foo int
// if foo {
//     return foo
// }
```
which can be activated with `path/to/the/package/foo=return(123)` so that it returns 123 when it triggers.

The failpoint here can still be put to sleep with the command `github.com/coreos/etcd/mvcc/backend/becoreCommit=sleep(5000)` (but right now the functional tester only does `path/to/failpoint=panic(...)`)